liblangutil/Exceptions.h

```diff
@@ -189,6 +189,11 @@ class Error: virtual public util::Exception
 		return "Error";
 	}
 
+    static std::optional<std::string> stringToSeverity(Severity _severity)
+	{
+		return _severity ? std::optional<std::string>{"Error, Warning, Info"} : std::nullopt;
+	}
```

Comment:
What this function is doing does not match its name. Its converting `Severity` to `string` but we need it the other way around. It's also not doing it correctly. For example it would convert `Severity::Error` to `"Error, Warning, Info"`.

Finally, I think the function should ignore letter case. `"error"`, `"Error"`, `"ERROR"` and `"ErRoR"` should all be converted to  `Severity::Error`. We might also want it to ignore leading and trailing whitespace to be more robust.

Reply:
Done ! 👍